packages/dds/sequence/src/test/sharedString.spec.ts

```diff
@@ -318,6 +319,47 @@ describe("SharedString", () => {
             sharedString2.connect(services2);
         });
 
+        it("interval consistency", () => {
```

Comment:
Not a big deal, but this doesn't match the grammatical title convention, like "can/should update interval".

Reply:
True.